app/assets/stylesheets/fae/modules/tables/_actions.scss

```diff
@@ -7,6 +7,10 @@ th {
     width: 120px
   }
 
+  &.-action-narrow {
```

Comment:
Is this a class you're using in a specific app? I thought we had documentation on the available classes you can use, but I'm not finding it now :(